File: codewars/Where my anagrams at/Where_my_anagrams_at.py
def anagrams(word, words):
    a = [0] * 26
    for c in word:
        a[ord(c) - 97] += 1
    ans = []
    for s in words:
        b = [0] * 26
        for c in s:
            b[ord(c) - 97] += 1
        p = 1
        for i in range(26):
            if a[i] != b[i]:
                p = 0
                break
        if p == 1:
            ans.append(s)
    return ans

File: codewars/Where my anagrams at/where my anagrams at.js
function anagrams(word, words) {
    let checking = function(splitedWord, splittedWordFromWords) {
        for (i1 in splitedWord) {
            if (splitedWord[i1] !== splittedWordFromWords[i1]) {
                return false;
            } 
        }
        return true;
    }
    let ansList = [];
    let splitedWord = {};
    word.split('').forEach(element => {
        splitedWord[element] = (splitedWord[element] || 0) + 1;
    });
    let splittedWordFromWords = {};
    for (let i = 0; i < words.length; i++) {
        if (words[i].length === word.length) {
        words[i].split('').forEach(element => {
            splittedWordFromWords[element] = (splittedWordFromWords[element] || 0) + 1;
        });
        if (checking(splitedWord, splittedWordFromWords)) {
            ansList.push(words[i]);
        }
        Object.keys(splittedWordFromWords).forEach(k => delete splittedWordFromWords[k])
    }
}
return ansList
}